doc/ref_model/chapters/chapter09.rst

```diff
@@ -134,7 +134,41 @@ Automation
 Infrastructure LCM Automation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To be covered in the next release.
+In the typical telecom operator environment, the infrastructure Life Cycle Management has become increasingly complex and error prone, with multiple vendors and products, and high maintenance costs due to the necessary planning, testing and the out-of-business-hours execution required to do frequently disruptive maintenance (e.g., upgrades) and to mitigate outages to mission-critical applications. Consistent process and tooling for infrastructure management across hybrid environments creates additional complexity due to different levels of access to infrastructure with the hands-on access in the on-premise operations and an access restricted to consumable services offered by public clouds.
```

Comment:
Some suggested changes.
```suggestion
In a typical telecom operator environment, infrastructure Life Cycle Management is highly complex and error-prone. The environment, with its multiple vendors and products, is maintenance expensive (both time and costs) because of the need for complex planning, testing, and the out-of-business-hours execution required to perform disruptive maintenance (e.g., upgrades) and to mitigate outages to mission-critical applications. Processes and tooling for infrastructure management across hybrid environments create additional complexity due to the different levels of access to infrastructure: hands-on access to the on-premise infrastructure but only restricted access to consumable services offered by public clouds.
```

Reply:
What is missing and needs to be added is the scope of LCM in on-premise and public clouds.
For example,
On-premise LCM: Infrastructure including BIOS; Host OS (including the kernel) and Tooling; Guest OS and Tooling; Tenant workloads
Public cloud LCM: Prepare for and act on public cloud infrastructure, Host OS and Tooling LCM; Guest OS and Tooling; Tenant workloads